### PR TITLE
docs(yard): adds yard to notification component page

### PR DESCRIPTION
**Pull request by nadiia**

### CODE REVIEW

File: documentation-site/components/yard/config/notification.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      placeholder: '() => {}',
+    },
+    autoHideDuration: {
+      value: undefined,
+      type: PropTypes.Number,
+      description: `The number of milliseconds to wait before automatically dismissing a
+        notification. This behavior is disabled when the value is set to 0.`,
+      placeholder: 10000,
+    },
+    ...pick(changeHandlers, ['onMouseEnter', 'onMouseLeave']),
```
Comment (gergelyke) [inline documentation-site/components/yard/config/notification.ts]: should we combine the two `pick`s from the changeHandlers?

Comment (nadiia) [reply documentation-site/components/yard/config/notification.ts]: it's broken for the look 🤷‍♀

**Comment by tajo** [comment]
Hm, do those style overrides actually do anything?

**Comment by nadiia** [comment]
> Hm, do those style overrides actually do anything?

Should be passed as overrides to the toast elements since it's reused.

**Pull request merged**